src/perception/CMakeLists.txt

```diff
@@ -19,12 +19,9 @@ aikido_check_package(std_msgs "perception" "std_msgs")
 find_package(visualization_msgs QUIET)
 aikido_check_package(visualization_msgs "perception" "visualization_msgs")
 
-# TODO: Workaround because find_package(tf) imports "tests" and "run_tests"
-# targets that conflict with the targets defined by Aikido.
-find_path(tf_INCLUDE_DIRS tf PATHS "/opt/ros/indigo/include"
-  PATH_SUFFIXES "interactive_markeres/interactive_marker_client.h")
```

Comment:
> Also, I have no idea why the interactive_marker_client.h path suffix is used here for TF.

I believe that was a mistake. [I fixed it in `Findtf.cmake`](https://github.com/personalrobotics/aikido/pull/198/files#diff-b9abb4ef14835558cc1d5aabdf1035fbR20).

> Do we need to put these path hints in the Findtf.cmake file? They are definitely good defaults.

In `Findtf.cmake`, I used `tf.pc` that includes better hint, which is generally correct.